supporter-product-data/src/main/scala/com/gu/services/SelectActiveRatePlansQuery.scala

```diff
@@ -0,0 +1,39 @@
+package com.gu.services
+
+import com.gu.model.FieldsToExport._
+import com.gu.model.Stage
+import com.gu.model.Stage.{DEV, PROD}
+
+import java.time.LocalDate
+
+object SelectActiveRatePlansQuery {
+
+  val name = "select-active-rate-plans"
+
+  val isNotDSGift = "Subscription.RedemptionCode__c = ''"
+  val isRedeemedDSGift = s"(Subscription.RedemptionCode__c != '' AND ${gifteeIdentityId.zuoraName} != '')"
+
+  def excludeDiscountProductRatePlans(discountProductRatePlanIds: List[String]) =
+    discountProductRatePlanIds
+      .map(discountProductRatePlanId => s"${productRatePlanId.zuoraName} != '$discountProductRatePlanId' AND")
+      .mkString("\n")
```

Comment:
just to make the query consistent below do this and add the AND to the end of line 34
```suggestion
      .map(discountProductRatePlanId => s"${productRatePlanId.zuoraName} != '$discountProductRatePlanId'")
      .mkString(" AND\n")
```